contracts/voting-verifier/src/events.rs

```diff
@@ -145,7 +145,7 @@ impl TryFrom<Message> for TxEventConfirmation {
             destination_address: other.destination_address,
             destination_chain: other.destination_chain,
             source_address: other.source_address,
-            payload_hash: other.payload_hash,
+            payload_hash: HexBinary::from(other.payload_hash),
```

Comment:
Why is this event `TxEventConfirmation` not changed?

Reply:
Didn't want to broaden the scope of the PR and didn't check what exactly this is. Let me see if it makes sense to include it into this PR